Auto Deposit on Hotkey (Default: B)

RolandSolymosi opened the pull request.

johulk committed:
Screen and draggable hud

johulk committed:
Merge remote-tracking branch 'origin/master'

johulk committed:
Screen and draggable hud

johulk committed:
Small task refactor ft Jeremyrab code

johulk committed:
Merge remote-tracking branch 'origin/dev' into dev

Commit:
Auto Deposit on Hotkey (Default: B)

Commit:
Added a scheduler to fix the special case of no deposit done. For now, only managed to do it with a timer.

johulk committed:
HUD Scale and Drag
Task List Sort and Warp
Task List Fixes for ticket tasks and nm world - credit to Jeremyrab
Auto-deposit - credit to Alvin_The_Fox

johulk reviewed (commented):
Approved, merged to dev branch manually, credits given.
Thank you for your contribution o7

Commit:
Reorganize the functions to modules

Commit:
Reorganize the functions to modules

Commit:
Moved Click to MenuInteractions too

Commit:
Moved Click to MenuInteractions too

johulk committed:
Fixed ticket task - Credit to Jeremyrab
Fixed Large Number (K/M) tasks

Commit:
First version of a ScreenInteraction handling solution, with it's builder and inner queued executor.

Commit:
First version of a ScreenInteraction handling solution, with it's builder and inner queued executor.

Commit:
Merge remote-tracking branch 'origin/reorg' into reorg

Commit:
Fixed command alias module to handle disconnect and join of servers.

Commit:
Initial merge of Task Collection with Task handling.
Also extended configuration to toggle swap wardrobe and hotbar slot based on number 1-5 (Combat, Mining, Foraging, Farming, Fishing)

Commit:
Remove duplicated Shiny text

Commit:
Small task collection fixes

Commit:
Config menu change (Currently still experimenting with 2 library)

Commit:
Update config menu.

Commit:
Fix Nightmare list a bit

Commit:
Add missing Hud Resize to option. Separate auto wardrobe from auto tool. Add fallback chain of teleport commands.

Commit:
Fix Gaming tasks commands

Commit:
Add Combat Fishing as a 6th wardrobe set

Commit:
Add Combat Fishing as a 6th wardrobe set

Commit:
Merge remote-tracking branch 'origin/reorg' into reorg

# Conflicts:
#	src/client/java/com/incrementalqol/config/Config.java

Commit:
Rework screen interaction handling to use low level packets.
Currently, not yet protected from multiple listener doing active interaction on the same screen (passive one shouldn't be an issue)

Commit:
Fix an index out of range issue in rare cases.

Commit:
Beta version read:
- Packet based interaction handling
- Auto skill
- Task refresh of prestige
- Task refresh on world change

Commit:
Fixes to the beta version.

Commit:
Version bump

Commit:
Change default scaling to 1.0

Commit:
Fix regex of level up text to match spear fishing because of whitespace.

Commit:
bump version

Commit:
Auto level up on next if no task remained.
Fix wrong config toggle triggers auto level up.

Commit:
Version beta4

Commit:
Warp automatically after read next set on auto level up.

Commit:
Fix settings names in config

Commit:
Handle cases of multiple digit long level (Capacity skills)

Commit:
Bump to beta5

Commit:
Fix sharpshooting (different menu slot because of ammo) and bump to beta 6

Commit:
Fix a few bug and update nightmare warps

Commit:
Fix tasks which name have regex special character names.

Commit:
Temporary fix for Ghost Inventory issue, till ConcurrentScreenHandler is made.

Commit:
Potion refresh enforcement

RolandSolymosi committed:
Update build.yml

Commit:
Add balloon rope attachment hiding option.

Commit:
Merge branch 'reorg'

# Conflicts:
#	src/client/resources/fabric.mod.json

Commit:
Merge remote-tracking branch 'origin/master'

Commit:
- [BIG] Rework Screen Handler for Queue logic
 - Add option to disable ropes of balloon of user
 - Add Highlighter experiment

Commit:
- [BIG][Ongoing] Rework Screen Handler for Queue logic
 - [Ongoing] Add Highlighter experiment
 - Add option to disable ropes of balloon of user
 - Update to 1.21.5 minecraft version

Commit:
- Merge in Jeremyrab's Task override feature

Commit:
Changes made by Jeremyrab preparing for World 4

Highlights
 - Added W4 tasks
 - Spotlighted tasks are now cyan in the HUD
 - Improved Quest and Tutorial tasks in HUD and replaced missing warp message for them
 - Disabled warping to next task during a boss fight in the boss arena dimension
 - Fixed default tools on skill specific tasks in NM
Other changes
 - Added toggle on and off for HUD
 - Added toggle to turn off HUD during boss fights
 - Replaced HUD background toggle with opacity slider
 - Auto skill module now gets the correct slot based on current progression
 - If Auto skill module encounters a skill that is not accessible, it skips over that skill as opposed to just stopping
 - Task HUD properly hides with F1
 - Skill points no longer allocated when switching between nightmare and normal realm, so task switching should be faster
 - Added option whether to immediately warp on auto level up
 - Added new Brush skill to auto skill module
 - Added 4 more loadout slots bound to numpad keys
 - Refresh tasks on more cases (ascension, etc.)
 - Added basic crop task
 - Improved some default tools/warps

Commit:
Fixed made by Jeremy for W4

Commit:
Restructure Phase 1: Data moved (W1 only, still need colored fish constraint)

Commit:
Merged Jeremyrab's pet exp and task coloring configuration.

Commit:
Merged Jeremyrab's Fixes for W4 2.5.1

Commit:
Merged Jeremyrab's Fixes for W4 2.5.2

Commit:
Create InsertableList for easier move of ListOptions.
Create ComplexTypeController, so sub screens can be used for more multi field sub types of configuration.

johulk committed:
Preview for 2.5.3 ->< Consumable HUD
